README.md

```diff
@@ -181,9 +181,9 @@ etags` generate tags for vi and Emacs respectively.  (Make sure that you are
 using the correct version of ctags; there are several competing programs with
 the same name in existence.)
 
-To enable building code documentation, please run
-`bootstrap.sh -DENABLE_DOCS=ON`. This enables the `make docs` rule to
-generate documentation. The HTML output is available in
+To enable building code documentation, please run `cmake
+.. -DENABLE_DOCS=ON`.  This enables the `make docs` rule to generate
```

Comment:
Works both with cmake and bootstrap.sh, as bootstrap passes the extra args to cmake :)

Reply:
I see, the way the README has been rewritten it looked like the bootstrap script is gone; it is never referenced anymore.
I see that it is still there.